src/components/Header.client.jsx

```diff
@@ -1,11 +1,11 @@
-import {useEffect, useState} from 'react';
+import {useEffect, useState, useContext} from 'react';
```

Comment:
Is that necessary? Can't see it used somewhere

Reply:
Remove most of unnecessary imports